lib/msf/jmx/discovery.rb

```diff
@@ -0,0 +1,26 @@
+# -*- coding: binary -*-
+
+module Msf
+  module Jmx
+    # This module provides methods which help to handle JMX end points discovery
+    module Discovery
+      # Builds a Rex::Java::Serialization::Model::Stream to discover
+      # an JMX RMI endpoint
+      #
+      # @return [Rex::Java::Serialization::Model::Stream]
+      def discovery_stream
+        block_data = Rex::Java::Serialization::Model::BlockData.new(
+          nil,
+          "\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00" +
+              "\x00\x00\x00\x02\x44\x15\x4d\xc9\xd4\xe6\x3b\xdf"
```

Comment:
The first padding of "\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00" is just a padding becuase there isnt an UnicastRef objid still... I'll document it, indeed will be useful.